Modules/System/Permission Sets/src/TenantPermissionSubForm.page.al

```diff
@@ -375,6 +644,46 @@ page 9859 "Tenant Permission Subform"
         CurrPage.Update(false);
     end;
 
+    local procedure UpdateSelected(RIMDX: Text[1]; PermissionOption: Option)
+    var
+        TenantPermission: Record "Tenant Permission";
+        OriginalTenantPermission: Record "Tenant Permission";
+    begin
+        CurrPage.SetSelectionFilter(TenantPermission);
+
+        if TenantPermission.FindSet() then
+            repeat
+                case RIMDX of
+                    'R':
+                        if TenantPermission."Object Type" = "Object Type"::"Table Data" then
+                            TenantPermission."Read Permission" := PermissionOption;
```

Comment:
in our code we often use a pattern to set a Update boolean and only run the modify if anything has changed.

Reply:
Makes sense. I do not see that it makes big impact here but I can certainly change it.

Reply:
done